sensecap-controller-backend/src/main/kotlin/com/inframousriddles/sensecap/routes/SenseCAPRoutes.kt

```diff
@@ -0,0 +1,9 @@
+package com.inframousriddles.sensecap.routes
+
+object SenseCAPRoutes {
+  const val ROOT = "/sensecap"
+  const val SHUTDOWN_ROUTE = "/shutdown"
+  const val REBOOT_ROUTE = "/reboot"
+  const val RESET_BLOCKS_ROUTE = "/reset-blocks"
+  const val FAST_SYNC_ROUTE = "/fast-sync"
```

Comment:
:mag: we can also move this to `HotspotAction` but no strong feelings

Reply:
@charbgr This is routes though 🤔